src/components/checkbox/index.tsx

```diff
@@ -195,20 +198,23 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
     }).start();
   }
 
+  setValidation(newValue: boolean) {
```

Comment:
I'd call it `testValidation`

Reply:
I think 'set' better describes what the func does